07-introduction-to-mockito/projects/final/app/build.gradle

```diff
@@ -73,4 +73,5 @@ dependencies {
   testImplementation 'junit:junit:4.13.2'
   testImplementation 'org.mockito.kotlin:mockito-kotlin:3.2.0'
   testImplementation 'androidx.arch.core:core-testing:2.1.0'
+  testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0'
```

Comment:
This is the old dependency that I updated [here](https://github.com/raywenderlich/atdd-materials/pull/2/commits/25daeebb6ec6e4660c730ea234f16834252bb767#diff-e07699b74b18c30cd47759a8bc4969eaa72693f8a864ca63375626b926a66f66R74).
So, `com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0` is not needed anymore.

I've run the tests in the final project and they all passed without any problems.